core/modules/simpletest/simpletest.pages.inc

```diff
@@ -308,6 +308,12 @@ function simpletest_settings_form($form, &$form_state) {
     '#description' => t('The verbose data will be printed along with the standard assertions and is useful for debugging. The verbose data will be erased between each test suite run. The verbose data output is very detailed and should only be used when debugging.'),
     '#default_value' => $config->get('simpletest_verbose'),
   );
+  $form['general']['simpletest_cache'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Cache instal profiles'),
```

Comment:
Typo: "instal" -> "install" ...or better use the full word "installation" instead:

```suggestion
    '#title' => t('Cache installation profiles'),
```